LocalDockerfile

```diff
@@ -0,0 +1,30 @@
+# Specify base image for building service binary
+FROM golang:1.20
```

Comment:
local.Dockerfile, production.Dockerfile, etc would be the preferred naming convention.  The `.Dockefile` extension is understood by github and most editors for syntax highlighting

Reply:
good call, changing